packages/azure-http-specs/specs/azure/resource-manager/resources/location.tsp

```diff
@@ -0,0 +1,174 @@
+        "createdBy": "AzureSDK",
+        "createdByType": "User",
+        "createdAt": <any date>,
+        "lastModifiedBy": "AzureSDK",
+        "lastModifiedAt": <any date>,
+        "lastModifiedByType": "User",
+      }
+    }
+    ```
+    """)
+  createOrUpdate is ArmResourceCreateOrReplaceSync<LocationResource>;
+
+  @scenario
+  @scenarioDoc("""
+    Resource PATCH operation.
+    Expected path: /subscriptions/00000000-0000-0000-0000-000000000000/providers/Azure.ResourceManager.Resources/locations/eastus/locationResources/resource
+    Expected query parameter: api-version=2023-12-01-preview
+    Expected request body:
+    ```json
+    {
+      "properties": {
+        "description": "valid2",
+      }
+    }
+    ```
+    Expected response body:
+    ```json
+    {
+      "id": "/subscriptions/00000000-0000-0000-0000-000000000000/providers/Azure.ResourceManager.Resources/locations/eastus/locationResources/resource",
+      "name": "resource",
+      "type": "Azure.ResourceManager.Resources/locationResources",
+      "properties":{
+        "description": "valid2",
+        "provisioningState": "Succeeded"
+      },
+      "systemData": {
+        "createdBy": "AzureSDK",
+        "createdByType": "User",
+        "createdAt": <any date>,
+        "lastModifiedBy": "AzureSDK",
+        "lastModifiedAt": <any date>,
+        "lastModifiedByType": "User",
+      }
+    }
+    ```
+    """)
+  update is ArmResourcePatchSync<LocationResource, LocationResourceProperties>;
+
+  @scenario
+  @scenarioDoc("""
+    Resource DELETE operation.
+    Expected path: /subscriptions/00000000-0000-0000-0000-000000000000/providers/Azure.ResourceManager.Resources/locations/eastus/locationResources/resource
+    Expected query parameter: api-version=2023-12-01-preview
+    Expected response status code: 204
+    """)
+  delete is ArmResourceDeleteSync<LocationResource>;
+
+  @scenario
+  @scenarioDoc("""
+    Resource List operation by location.
+    Expected path: /subscriptions/00000000-0000-0000-0000-000000000000/providers/Azure.ResourceManager.Resources/locations/eastus/locationResources
+    Expected query parameter: api-version=2023-12-01-preview
+    
+    Expected response body:
+    ```json
+    {
+      "value": [{
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/providers/Azure.ResourceManager.Resources/locations/eastus/locationResources/resource",
+        "name": "resource",
+        "type": "Azure.ResourceManager.Resources/locationResources",
+        "properties":{
+          "description": "valid",
+          "provisioningState": "Succeeded"
+        },
+        "systemData": {
+          "createdBy": "AzureSDK",
+          "createdByType": "User",
+          "createdAt": <any date>,
+          "lastModifiedBy": "AzureSDK",
+          "lastModifiedAt": <any date>,
+          "lastModifiedByType": "User",
+        }
+      }]
+    }
+    ```
+    """)
+  listByParent is ArmResourceListByParent<LocationResource>;
```

Comment:
listByLocation

Reply:
Fixed it by the [commit#65121db](https://github.com/Azure/typespec-azure/pull/1932/commits/65121db86d731e7610872fe5daf21420bedd551a)